packages/babel-plugin-remove-graphql-queries/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -54,6 +54,17 @@ export default (() => React.createElement(StaticQuery, {
 }));"
 `;
 
+exports[`Transforms queries defined in own variable in useStaticQuery 1`] = `
+"import staticQueryData from \\"public/static/d/2626356014.json\\";
+import React from 'react';
+import { useStaticQuery } from 'gatsby';
+export default (() => {
+  const query = \\"2626356014\\";
```

Comment:
Need to remove query as well in babel transform in case of queries declared in their own variables

Reply:
@DSchau Any reason you can think of to keep these declarations?

Reply:
Yeah - I was actually going to comment on this. Again--I'd say it's sort of outside the scope of this PR, but they seem... useless, right?

Reply:
Definitely useless for both useStaticQuery as well as StaticQuery. I'll go ahead and remove in a later PR!

Definitely some cleanup to be done